tests/session/transaction-integration_error-001.phpt

```diff
@@ -0,0 +1,70 @@
+--TEST--
+MongoDB\Driver\Session: Setting per-op readConcern or writeConcern in transaction
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_not_replica_set(); ?>
+<?php skip_if_server_version('<', '4.0'); ?>
+<?php skip_if_not_clean(DATABASE_NAME, COLLECTION_NAME); ?>
```

Comment:
`skip_if_not_clean()`. No need to use arguments unless you're customizing the defaults (e.g. custom collection).

Reply:
Fixed.